packages/devextreme/js/events/events.types.d.ts

```diff
@@ -1,7 +1,92 @@
+type PickFromEvent<TNativeEvent extends Event, TTargetEvents extends Event, PropName extends keyof TTargetEvents> =
+  TNativeEvent extends Pick<TTargetEvents, PropName>
+    ? TTargetEvents[PropName]
+    : undefined;
```

Comment:
I guess, it's meant to be `never`?

```suggestion
    : never;
```

Reply:
`undefined` is better
`never` is assignable to any other type, it will hide errors
https://www.typescriptlang.org/play/?#code/CYUwxgNghgTiAEYD2A7AzgF3lAXPFIAbiDANwBQ5y6WARnpjAJYoDm8AvNhdZonigCuAW1olO3ckA